api/database/migrations/exif_invalid_gps_test.go

```diff
@@ -0,0 +1,74 @@
+package migrations_test
+
+import (
+	"os"
+	"math"
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/photoview/photoview/api/database/migrations"
+	"github.com/photoview/photoview/api/graphql/models"
+	"github.com/photoview/photoview/api/test_utils"
+)
+
+func TestExifMigration(t *testing.T) {
+  envFile, err := os.Open("/home/runner/work/photoview/photoview/api/testing.env")
+	if err != nil {
+	  t.Fatalf("Failed to open environment file: %v", err)
+	}
+	defer envFile.Close()
+
+	scanner := bufio.NewScanner(envFile)
+	for scanner.Scan() {
+		line := scanner.Text()
+		parts := strings.SplitN(line, "=", 2)
+		if len(parts) != 2 {
+		  t.Fatalf("Invalid line in environment file: %s", line)
+		}
+		key, value := parts[0], strings.Trim(parts[1], "'")
+		os.Setenv(key, value)
+	}
+
+	db := test_utils.DatabaseTest(t)
+	//defer db.Exec("DELETE FROM media_exif") // Clean up after test
```

Comment:
Should we uncomment this?

Reply:
I'm not sure about the cleanup here, as having media with GPS data in the testing DB doesn't break other tests.
In general, I know that it is better to generate and clean data for each test, I wasn't sure about the current tests suite execution, so I didn't enable the cleanup.
If you confirm that it is better to do this, I'll uncomment the line.

Reply:
I hope to clean the test data after the test. It's friendly to run tests locally.

Reply:
ok, I'll uncomment the cleanup after the workflows are enabled in the repo, as now they got disabled for some reason(